### PR TITLE
fix(auto): Add help line at the end of a scan with fingerprint information when using config auto

### LINKED ISSUE #4674: Semgrep prints a non-zero findings count but shows no results
**Describe the bug**
`semgrep --config auto .` on some repos will show a findings count but no findings.

**Expected behavior**
Findings count should match number of findings

**Screenshots**
```
Auto config uses Semgrep rules to scan your codebase and the Semgrep Registry to generate recommended rules based on
your languages and frameworks.
Fetching rules from https://semgrep.dev/registry ...
Running 191 rules...
100%|██████████████████████████████████████████████████|6/6 files

skipped: all .gitignored files, 2 files matching .semgrepignore patterns
for a detailed list of skipped files, run semgrep with the --verbose flag

ran 191 rules on 4 files: 5 findings
```

**What is the priority of the bug to you?**

- [ ] P0: blocking your adoption of Semgrep or workflow
- [ ] P1: important to fix or quite annoying
- [x] P2: regular bug that should get fixed

**Environment**
CLI v 0.82.0

### PR DESCRIPTION
**Pull request by minusworld**
Fixes https://github.com/returntocorp/semgrep/issues/4674.

PR checklist:

- [x] Documentation is up-to-date
- [x] Changelog is up-to-date
- [x] Change has no security implications (otherwise, ping security team)

### CODE REVIEW

**Comment by minusworld** [comment]
Output looks like this.

With fingerprints:
```
Fetching rules from https://semgrep.dev/registry ...
Running 56 rules...
100%|████986/986 files

skipped: all .gitignored files, 2 files larger than 1.0 MB, 50 files matching .semgrepignore patterns
for a detailed list of skipped files, run semgrep with the --verbose flag

ran 56 rules on 718 files: 0 findings
(374 fingerprint findings. Run --config auto again in a few seconds to run new recommendations)
```

Without fingerprints:
```
[ ... ]
skipped: all .gitignored files, 2 files larger than 1.0 MB, 50 files matching .semgrepignore patterns
for a detailed list of skipped files, run semgrep with the --verbose flag

ran 135 rules on 268 files: 2 findings
```

File: semgrep/semgrep/output.py [hunk 1]
```diff
@@ -300,15 +301,22 @@ def output(
...
+                    lambda m: m.severity == RuleSeverity.INVENTORY, self.rule_matches
+                )
+                num_findings = len(regular_matches)
+                num_fingerprint_findings = len(fingerprint_matches)
                 num_targets = len(self.all_targets)
                 num_rules = len(self.filtered_rules)
 
                 ignores_line = str(ignore_log or "No ignore information available")
                 stats_line = f"ran {num_rules} rules on {num_targets} files: {num_findings} findings"
+                auto_line = f"({num_fingerprint_findings} fingerprint findings. Run --config auto again in a few seconds to run new recommendations)"
```
Comment (ievans) [inline semgrep/semgrep/output.py]:
```suggestion
                auto_line = f"({num_fingerprint_findings} fingerprint findings. Run --config auto again in a few seconds use new rule recommendations)"
```

**Review by ievans — CHANGES_REQUESTED** [review]
Can we have a test for this at all @minusworld ?

**Comment by minusworld** [comment]
@ievans Added one, lmk what you think

**Review by ievans — APPROVED** [review]
looks great! Thanks @minusworld!

**Pull request merged**